ipaserver/install/server/replicainstall.py

```diff
@@ -770,6 +770,15 @@ def promotion_check_ipa_domain(master_ldap_conn, basedn):
         ))
 
 
+def promotion_check_host_principal_auth_ind(master_ldap_conn, hostdn):
+    entry = master_ldap_conn.get_entry(hostdn, ['krbprincipalauthind'])
```

Comment:
We are trying to get away from the word master. I think just conn is fine.

Reply:
Otherwise this is fine. IMHO you can make this change and drop the temp commit at the same time.

LGTM.